src/chat-client/index.ts

```diff
@@ -48,6 +48,7 @@ export interface ChatSendOptions {
   membersType?: chat1.ConversationMembersType
   confirmLumenSend?: boolean
   replyTo?: chat1.MessageID
+  explodingLifetime?: string
```

Comment:
Thanks for the PR! I think `explodingLifetime` should be a number here (same for the other one), as in millisecond, just like how the duration is specified in other JS apis like `setTimeout`. Then before sending it to the CLI api, we just need to add the `ms` so they can be properly [parsed](https://golang.org/pkg/time/#ParseDuration) on the [Go side](https://github.com/keybase/client/blob/b9875b85992d7af0573cab8afb6aa46f7227997a/go/client/chat_api_handler.go#L167).